### PR TITLE
feat/add swagger for api

### PR DESCRIPTION
**Pull request by JoE11-y**
For #49 

Work done:
1. Update dashboard route documentation
2. Update user route documentation
3. Update positions route documentation

Work to be done:
Optimizations

### CODE REVIEW

File: web_app/api/serializers/user.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from pydantic import BaseModel, Field
+
+class CheckUserResponse(BaseModel):
+    is_contract_deployed: bool = Field(..., example=False, description="Indicates if the user's contract is deployed.")
+
+
+class UpdateUserContractResponse(BaseModel):
+    is_contract_deployed: bool = Field(..., example=False, description="Indicates if the user's contract is deployed.")
+
+class GetUserContractAddressResponse(BaseModel):
```
Comment (djeck1432) [inline web_app/api/serializers/user.py:10]: Please, add blank line here to keep formatting

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Looks good to me, just one issue with formatting

File: web_app/api/serializers/user.py (hunk 1 above)
Comment (JoE11-y) [reply web_app/api/serializers/user.py:10]: oh okay.

**Review by djeck1432 — APPROVED** [review]
looks good to me

**Pull request merged**